src/main/java/fi/okm/jod/yksilo/service/profiili/Mapper.java

```diff
@@ -92,6 +92,21 @@ public static KoulutusDto mapKoulutus(Koulutus entity) {
             entity.getOsasuoritukset());
   }
 
+  public static KoulutusDto mapKoulutusForPoll(Koulutus entity) {
+    return entity == null
+        ? null
+        : new KoulutusDto(
+            entity.getId(),
+            null,
+            null,
+            null,
+            null,
+            extractOsaamisetUris(entity),
+            isOsaamisetOdottaaTunnistusta(entity.getOsaamisenTunnistusStatus()),
+            isOsaamisetTunnistusEpaonnistui(entity.getOsaamisenTunnistusStatus()),
+            null);
+  }
```

Comment:
Either create a separate Dto for this special case or use the existing mapper. E.g. setting nimi to null violates the OpenAPI spec which promises that nimi is always present.